lib/googleHelpers.js

```diff
@@ -0,0 +1,60 @@
+const { OAuth2Client } = require('google-auth-library')
```

Comment:
Could you please add unit tests for these new functions?

Reply:
Done!